### PR TITLE
Fixed mobile view for create skill

### LINKED ISSUE #2212: Separate privacy and terms page
**Actual Behaviour**
Currently the privacy and terms page are the same.

**Expected Behaviour**
Have separate privacy and terms page.
**Screenshots of the issue**
Screenshot 2019-06-06 at 4 23 23 PM
Screenshot 2019-06-06 at 4 23 15 PM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2212

Changes: Fixed mobile view for create skill

Demo Link: https://pr-2405-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
ple

### CODE REVIEW

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 1]
```diff
@@ -54,13 +54,40 @@ let languages = [];
 const Paper = styled(_Paper)`
   width: 100%;
   margin-top: 1.25rem;
+  // display: flex;
```
Comment (fragm3) [inline src/components/cms/SkillCreator/SkillWizard.js]: Remove comment

**Comment by akshatnitd** [comment]
Notice the spacing, sizes and gaps carefully.

**Comment by samagragupta** [comment]
@akshatnitd Made the required changes
@fragm3 
Capture

**Comment by samagragupta** [comment]
@akshatnitd 
Capture

**Comment by harsh-jindal** [comment]
Approved as work in progress. Will take up in the remaining changes in #2409

**Comment by mariobehling** [comment]
Thank you. Please try to resolve issue in a much shorter amount of time to avoid that PRs become blockers.

**Pull request merged**